README.adoc

```diff
@@ -1069,11 +1069,17 @@ User.where("created_at > :start AND created_at < end", start: 30.days.ago, end:
 User.where(created_at: 30.days.ago..7.days.ago)
```

Comment:
Should we also change the above bad examples? They are exclusive on both ends.

Reply:
Good catch. Updated with b720777